src/components/organisms/UiMultipleChoices/UiMultipleChoices.stories.ts

```diff
@@ -0,0 +1,333 @@
+import { ref } from 'vue';
+import type {
+  Meta,
+  StoryObj,
+} from '@storybook/vue3';
+import { useArgTypes } from '@sb/helpers';
+import {
+  UiAlert,
+  UiBulletPoints,
+  UiHeading,
+  UiMultipleChoices,
+  UiSidePanel,
+  UiText,
+} from '@index';
+import UiMultipleChoicesItem from '@/components/organisms/UiMultipleChoices/_internal/UiMultipleChoicesItem.vue';
+import type { MultipleChoicesProps } from '@index';
+import './UiMultipleChoices.scss';
+
+type MultipleChoicesArgsType = MultipleChoicesProps;
+type MultipleChoicesMetaType = Meta<MultipleChoicesArgsType>;
+type MultipleChoicesStoryType = StoryObj<MultipleChoicesArgsType>;
+
+const { argTypes } = useArgTypes(UiMultipleChoices);
+const isSidePanelOpen = ref(false);
+
+function handleButtonInfoClick() {
+  isSidePanelOpen.value = !isSidePanelOpen.value;
+}
+
+const buttonInfoItems = [
```

Comment:
why do you duplicate items? you have items for button info (used once) and items in `meta`. You can define items and extend it (use .map) in stories args